imagetest/test_suites/ssh/image_ssh_test.go

```diff
@@ -67,13 +71,37 @@ func createSession(user, host string, pembytes []byte) (*ssh.Client, *ssh.Sessio
 
 	client, err := ssh.Dial("tcp", host, sshConfig)
 	if err != nil {
-		return nil, nil, err
+		return nil, err
 	}
+	return client, nil
+}
 
+// checkLocalUser test that the user account exists in /etc/passwd
+func checkLocalUser(client *ssh.Client, user string) error {
 	session, err := client.NewSession()
 	if err != nil {
 		client.Close()
```

Comment:
If you close the client, all tests after it would likely fail as the client wouldn't usable. It'll also likely hit "failed to close client" error as it'll already be closed.

I'd suggest removing client.Close() from checkLocalUser and checkSudoGroup, so instead it only returns that this tests failed and returned an error.

Reply:
Remove this line.